src/RpcClient/RpcClient.csproj

```diff
@@ -1,20 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Version>3.0.0-preview3</Version>
-    <TargetFramework>netstandard2.1</TargetFramework>
-    <RootNamespace>Neo.Network.RPC</RootNamespace>
```

Comment:
I think this line should be kept. `RpcClient` is not a plugin.

Reply:
That's why it doesn't have "Plugin" in the package name

Reply:
And the root namespace should also be different.